CMSIS/RTOS2/FreeRTOS/Examples/Examples.csolution.yml

```diff
@@ -0,0 +1,117 @@
+solution:
+  created-for: CMSIS-Toolbox@2.0.0
+
+  cdefault:
+
+  packs:
+    - pack: ARM::CMSIS@>=6.0.0-0
+    - pack: ARM::Cortex_DFP
+
+  target-types:
+    - type: CM0
+      device: ARMCM0
+      variables:
+        - Target-Layer: ../../Target/CM0/Target.clayer.yml
```

Comment:
These relative paths look weird. Shouldn't they be relartive to the csolution file?

Reply:
I don't know of any alternative way of specifying this given the solution structure.

Reply:
Sorry, I was mislead.